src/app/_components/Card.tsx

```diff
@@ -90,7 +90,7 @@ export function Card({
         )}
 
         {title && typeof title === 'string' ? (
-          <Heading tag="h3" variant="lg" className="line-clamp-2 text-ellipsis">
+          <Heading tag="h3" variant="lg">
```

Comment:
Should we add a prop `isClamped`? Since we've now effectively stopped clamping headings for blog post, I believe.